pkg/reconciler/taskrun/taskrun.go

```diff
@@ -743,19 +743,12 @@ func (c *Reconciler) createPod(ctx context.Context, ts *v1beta1.TaskSpec, tr *v1
 		logger.Errorf("Failed to create a pod for taskrun: %s due to output image resource error %v", tr.Name, err)
 		return nil, err
 	}
-
 	ts, err = resources.AddInputResource(ctx, c.KubeClientSet, c.Images, rtr.TaskName, ts, tr, inputResources)
 	if err != nil {
 		logger.Errorf("Failed to create a pod for taskrun: %s due to input resource error %v", tr.Name, err)
 		return nil, err
 	}
 
-	ts, err = resources.AddOutputResources(ctx, c.KubeClientSet, c.Images, rtr.TaskName, ts, tr, outputResources)
```

Comment:
why are we deleting `AddOutputResources` but not `AddInputResource`?At the same time, `ApplyResources` on L761 ⬇️ is as is.

Reply:
I think the reason is that we've removed all the allowedOutput types here in this PR but still Image resource could be the left input resources according to https://github.com/tektoncd/pipeline/pull/6150#discussion_r1105034369. But these should be all targeted to be removed in https://github.com/tektoncd/pipeline/pull/6167. 

Sorry for the confusion it might led to in this PR. I would love to follow if there're route that could make this more explicit/ less confusing 😇

Reply:
I think this has been addressed now? Both Output and Input are removed?